en/core-libraries/email.rst

```diff
@@ -483,7 +483,7 @@ following::
         {
             $this
                 ->to($user->email)
-                ->subject(sprintf('Welcome %s', $user->name))
+                ->subject(__('Welcome %s', $user->name))
```

Comment:
No, because that part specifically says how you can keep BC:

    return Package('sprintf', 'fallback_domain', $messages);

But the default is not sprintf unfortunately, so here in your changes it is indeed still {0}